File: atcoder/abc397a_thermometer.f
! Vicfred
! https://atcoder.jp/contests/abc397/tasks/abc397_a
! implementation
program main
  implicit none
  real :: X
  
  read(*,*) X
  
  if (X >= 38.0) then
    print *, 1
  else if (X >= 37.5) then
    print *, 2
  else
    print *, 3
  end if
  
end program main

File: atcoder/abc397a_thermometer.pas
(* vicfred
   https://atcoder.jp/contests/abc397/tasks/abc397_a
   implementation *)
program TemperatureCategory;

var
  X: Real;
  
begin
  (* Read input *)
  Readln(X);
  
  (* Apply condition logic *)
  if X >= 38.0 then
    Writeln(1)
  else if X >= 37.5 then
    Writeln(2)
  else
    Writeln(3);
end.
